ingress/istio-mesh/loadbalancer/install.sh

```diff
@@ -9,15 +9,26 @@ fi
 echo Operator init
 istioctl operator init
 
-echo Create ingress gateways and load balancers
-kubectl apply -f iop.yaml
+function installing_istio() {
+  echo Create ingress gateways and load balancers
+  kubectl apply -f iop.yaml
 
-echo Wait for all resources to come up
-sleep 30
-kubectl -n istio-system rollout status deploy istiod
-kubectl -n istio-system rollout status deploy istio-ingressgateway
-kubectl -n istio-system rollout status deploy istio-ingressgateway-internal
+  echo Wait for all resources to come up
+  sleep 30
+  kubectl -n istio-system rollout status deploy istiod
```

Comment:
can we set the namespace as env variable instead of hardcoding

Reply:
updated.